### PR TITLE
Add a Readme to introduce this repository

### LINKED ISSUE #1: Add a Readme
The Readme should indicate the purpose of the project and how to use it

We should also link the design document in the Readme https://docs.google.com/document/d/14oJux-d-91Do3DLi5eRG-wUEE4R3Hh7D4u0Yfje3auw/edit#heading=h.9chr39ggpwbe

### PR DESCRIPTION
**Pull request by lonelyCZ**
Signed-off-by: lonelyCZ <531187475@qq.com>

Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+### Configure Terraform Plugin Cache.
+
+The configuration directory of each dependence can share the cache to improve the execution speed.
+
+```
+export TF_PLUGIN_CACHE_DIR="$HOME/.terraform.d/plugin-cache" 
+```
+
+## Usage Steps
```
Comment (irbekrm) [inline README.md]: Suggestion: remove this part as at the moment we do not actually have that funcitonality in this repository.

That can be added together with the PR that actually adds Vault I think.

**Review by irbekrm — CHANGES_REQUESTED** [review]
Thank you @lonelyCZ !

This all looks good to me, except for I think that we should not add a section on Vault yet as it is not yet in the repository.

**Comment by lonelyCZ** [comment]
I will fill this readme one by one in next PRs.

/cc @irbekrm

**Comment by irbekrm** [comment]
Thank you @lonelyCZ !

Nice to get the first PR merged 🥳 

/lgtm
/approve

**Pull request merged**